modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/DropdownMenuTag.java

```diff
@@ -27,6 +27,15 @@
  */
 public class DropdownMenuTag extends ButtonTag {
 
+	@Override
+	public int doEndTag() throws JspException {
+		if (DropdownItemListUtil.isEmpty(_dropdownItems)) {
```

Comment:
Just as a minor improvement. 

We could store this value in a property in the doStartTag and then check the property in the doEndTag to avoid calling the Util two times.

Reply:
Thanks @javierdearcos! Added this in [ed00939](https://github.com/liferay-frontend/liferay-portal/pull/1936/commits/ed009399777b25922f206de8b2469bb9e34c887d)